tests/dns/dns-rcode/test.rules

```diff
@@ -0,0 +1,11 @@
+# Should alert in both directions as no flow is provided.
+alert dns any any -> any any (dns.rcode:3; sid:1; rev:1;)
+
+# Should only alert in the request direction.
+alert dns any any -> any any (dns.rcode:3; flow:to_server; sid:2; rev:1;)
```

Comment:
This should not alert at all. As an rcode of 3 in the to server direction does not exist in this pcap.